Datahub.Web/Env.cs

```diff
@@ -22,23 +22,23 @@ public class Env : IEnv
         public string ES_ENDPOINT_SCHEME { get; private set; }
         public string ES_ENDPOINT_HOST { get; private set; }
         public string ES_ENDPOINT_PORT { get; private set; }
-        public string ES_AWS_ACCESSKEY { get; private set; }
-        public string ES_AWS_SECRETACCESSKEY { get; private set; }
-        public string ES_AWS_REGION { get; private set; }
-        public string ES_AWS_PROFILE { get; private set; }
+        public string AWS_ACCESS_KEY_ID { get; private set; }
+        public string AWS_SECRET_ACCESS_KEY { get; private set; }
+        public string AWS_DEFAULT_REGION { get; private set; }
         public string GTM_ID { get; private set; }
+        public string DB_TABLE { get; private set; }
```

Comment:
Tip from pain experienced: You will need to add these new env vars on Beanstalk *before* you deploy it. Otherwise Beanstalk seems to get itself in a horrible state!

Reply:
Added DB_TABLE and AWS_DEFAULT_REGION to the beta and live env with appropriate values. I've left ES_AWS_REGION in place because I don't want to break stuff. This has been replaced by the standard AWS_DEFAULT_REGION environment var in this code.